### ISSUE TITLE
[Netlify/docs-tockosorg] Configuration error

### ISSUE THREAD

**Issue by niklasad1** [helena-project/tock#798]
The Netlify configuration seems to be broken:

``` bash
1:15:46 AM: error: intermittent IO error while iterating directory `/opt/buildhome/.rustup/toolchains/nightly-2018-01-05-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin`
1:15:46 AM: caused by: IO error for operation on /opt/buildhome/.rustup/toolchains/nightly-2018-01-05-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin: No such file or directory (os error 2)
1:15:46 AM: caused by: No such file or directory (os error 2)
1:15:46 AM: note: run with `RUST_BACKTRACE=1` for a backtrace
1:15:46 AM: make: *** [doc] Error 1
```

**Comment by ppannuto** [comment]
Ugh.., looks like something changed upstream. I can confirm this breaks locally if you update to xargo 0.3.11, which was just released. Looking into a fix now

**Comment by ppannuto** [comment]
Fixed by #799
